<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReturnBookWindow</class>
 <widget class="QMainWindow" name="ReturnBookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="rdb_bookid">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>书号</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_bookname">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>书名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_bookid">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_bookname">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>归还</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
